### ISSUE TITLE
URL encoded names in Nemo file browser

### ISSUE THREAD

**Issue by bozonx** [OpenMarshal/npm-WebDAV-Server#66]
That's a great packet, thanks for it! But there is some unexpected behavior.

I started server and after that connected via Nemo and it showed url encoded names of files like:
`file%20with%20spaces.txt` but actually file has this name `file with spaces.txt`.

Server code:

```
const webdav = require('webdav-server').v2;

const server = new webdav.WebDAVServer({
  port: 1900
});

server.setFileSystem('/somedir', new webdav.PhysicalFileSystem('/home/iam/somedir'), (success) => {
  server.start(() => console.log('READY'));
})
```

OS: Linux Mint 19.2 Cinnamon
Nemo 4.2.3

I didn't find any suitable server props in the wiki page https://github.com/OpenMarshal/npm-WebDAV-Server/wiki/Server's-options-and-customizations-%5Bv2%5D

**Comment by AdrienCastex** [comment]
Hello,

The problem was the encoding in the `displayName` tag of a PROPFIND method. I fixed it, but I am currently testing on different webdav clients on different OS to be sure everything is working as expected.

The fix will be in the version `2.5.2` (if everything is fine, the release will be tomorrow on GitHub and NPM). I will let you know here as soon as it is available.

Thank you very much for your contribution.

**Comment by AdrienCastex** [comment]
Hello,

The version `2.5.2` is released with the fix. Let me know by reopening this issue if it didn't solve the problem you encountered.
